ycmd.el

```diff
@@ -738,7 +738,7 @@ To see what the returned structure looks like, you can use
          content
          :parser 'json-read)))))
 
-(defun ycmd--handle-exception (results &optional default-handler)
+(defun ycmd--handle-exception (results)
```

Comment:
I removed the `default-handler` because we check for all types of exceptions here